zipkin-tracer.gemspec

```diff
@@ -18,30 +18,34 @@ $:.unshift lib unless $:.include?(lib)
 require 'zipkin-tracer/version'
 
 Gem::Specification.new do |s|
-  s.name                      = "zipkin-tracer"
+  s.name                      = 'zipkin-tracer'
   s.version                   = ZipkinTracer::VERSION
-  s.authors                   = ["Franklin Hu", "R Tyler Croy", "James Way"]
-  s.email                     = ["franklin@twitter.com", 'tyler@monkeypox.org', 'jamescway@gmail.com']
+  s.authors                   = ['Franklin Hu', 'R Tyler Croy', 'James Way', 'Jordi Polo', 'Julien Feltesse', 'Scott Steeg']
+  s.email                     = ['franklin@twitter.com', 'tyler@monkeypox.org', 'jamescway@gmail.com', 'jcarres@mdsol.com', 'jfeltesse@mdsol.com', 'ssteeg@mdsol.com']
   s.homepage                  = 'https://github.com/openzipkin/zipkin-tracer'
-  s.summary                   = "Ruby tracing via Zipkin"
-  s.description               = "Adds tracing instrumentation for ruby applications"
+  s.summary                   = 'Ruby tracing via Zipkin'
+  s.description               = 'Adds tracing instrumentation for ruby applications'
 
-  s.required_rubygems_version = ">= 1.3.5"
+  s.required_rubygems_version = '>= 1.3.5'
 
-  s.files                     = Dir.glob("{bin,lib}/**/*")
+  s.files                     = Dir.glob('{bin,lib}/**/*')
   s.require_path              = 'lib'
 
-  s.add_dependency "finagle-thrift", "~> 1.4.1"
-  s.add_dependency "rack", "~> 1.3"
+  s.add_dependency 'thrift'
```

Comment:
Is this a hard dependency now? Probably we can leave this out for peope who wants to only use json